apps/web/src/components/(dashboard)/layout/desktop-nav.tsx

```diff
@@ -25,13 +25,21 @@ const navigationLinks = [
   },
 ];
 
-export type DesktopNavProps = HTMLAttributes<HTMLDivElement>;
+export type DesktopNavProps = HTMLAttributes<HTMLDivElement> & {
+  isCommandMenuOpen?: boolean;
+  setIsCommandMenuOpen?: (value: boolean) => void;
```

Comment:
I think it's fine to just force these to not be undefined since this component isn't reused (and isn't planned to be reused) anywhere else

This would let us clean up the code a little and remove the open state here and get rid of the CommandMenu